Dockerfile

```diff
@@ -1,11 +1,103 @@
-FROM nfnty/arch-npm
+FROM opensuse:tumbleweed
```

Comment:
To be honest, I also looked at Fedora and Ubuntu and I originally wanted to go with openSUSE Leap instead of Tumbleweed but in all cases we would have needed to compile packages (Fedora and Leap) or rely on unofficial Repositories/PPAs (Ubuntu/Debian).

And honestly Arch was a terrible idea for many reasons :p